### PR TITLE
Set up vacation bot & enable it for the next two weeks

### PR DESCRIPTION
**Pull request by BenHenning**
## Explanation

This PR sets up a [vacation bot](https://devbots.xyz/documentation/vacation/) so that at times when we don't have hand-over plans for core team members, we can provide a general notice to the community that there may be delays. Since I'm going to be unavailable for the next two weeks, I'm enabling this until 6 December (#3998 is tracking disabling it).

Note that the validation that this was set up correctly will be done by verifying the message starts showing up tomorrow (I enabled it a day early for this reason).

## Essential Checklist

- [ ] ~The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)~
- [X] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [X] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only
N/A -- this is setting up a bot.

### CODE REVIEW

**Comment by BenHenning** [comment]
PTAL @seanlip

File: .devbots/vacation.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+enabled: true
+pull-comment: >
+    Thanks for submitting this pull request! Some main reviewers
+    have taken time off for the next few weeks, so it may take a
+    little while before we can look at this PR. We appreciate your
+    patience while some of our team members recharge.
+issue-comment: >
+    Thanks for filing this issue! Please note that some team members
+    are currently taking time off to recharge, so it might be a
+    while before we respond or triage this issue.
```
Comment (seanlip) [inline .devbots/vacation.yml]: respond --> respond to

**Review by seanlip — APPROVED** [review]
Looks fine, thanks -- I'm not sure whether you need the issue comment message, tbh (normally other folks triage issues, right?), but up to you.

File: .devbots/vacation.yml (hunk 1 above)
Comment (BenHenning) [reply .devbots/vacation.yml]: Nice catch--done.

**Comment by BenHenning** [comment]
Thanks @seanlip!

> Looks fine, thanks -- I'm not sure whether you need the issue comment message, tbh (normally other folks triage issues, right?), but up to you.

Not always. Since we don't have an established triage process yet, this is partly a stop-gap since I've been making a hint to follow up with issues when necessary over the last few weeks.

**Comment by seanlip** [comment]
OK! Sg, your call.

**Pull request merged**